.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
+
+
+.vite/
```

Comment:
I can see you tried to git-ignore the `vite` folder, but it doesn't work. I was thinking since this folder doesn't affect much your application, maybe try to delete and remove it from your folder structure.